examples/python/sourcepkg/user.py

```diff
@@ -0,0 +1,13 @@
+import sys
+sys.path.append('.')
```

Comment:
Hmm... is there any way we can avoid making the user do this? Can the runtime env server.py do this?